config/passport.js

```diff
@@ -9,36 +8,35 @@ module.exports = function(passport) {
 
   passport.deserializeUser(function(id, done) {
     User.findById(id, function(err, user) {
-        done(err, user);
+      done(err, user);
     });
   });
 
   passport.use(new GoogleStrategy({
-    clientID       : configAuth.googleAuth.clientID,
-    clientSecret   : configAuth.googleAuth.clientSecret,
-    callbackURL    : configAuth.googleAuth.callbackURL,
+    clientID     : configAuth.googleAuth.clientID,
```

Comment:
These may be variables that you want to store in `process.env` in order to avoid committing sensitive information to your public repository.